src/Keboola/S3Extractor/S3ExceptionConverter.php

```diff
@@ -18,6 +18,9 @@ public static function resolve(S3Exception $e): void
             case 403:
                 throw new Exception('Invalid credentials or permissions.', $e->getCode(), $e);
                 break;
+            case 503:
+                throw new Exception('Error 503 Slow Down: The number of requests to the S3 bucket is very high.', $e->getCode(), $e);
```

Comment:
no a ted vyvstava zajimava otazka - je kazy 503 error z S3 Slow Down ? a odpoved je takova neurcita
- https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html - neni, nicmene na oba pripady je remedy stejna - asi by to teda melo vyhazovat `$e->getMessage()` ?

Reply:
Good point, toho jsem si všiml až při řešení dalšího PR. Naštěstí dle té dokumentace na to mají vlastí code který je dostupný přes `getAwsErrorCode()`. Takže jsem ho uložil do konstanty a přidal podmínku.

Reply:
jo, to zni dobre, akorat me teda nenapadlo, ze jde napsat neco jako 
`case 503 && $e->getAwsErrorCode() === self::ERROR_CODE_SLOW_DOWN:` :)

Reply:
To byla blbost, promiň. Už jsem to opravil.

Reply:
to je v pohode, me uplne fascinuje, ze je neco takovyho syntakticky spravne a uz vubec me neni jasny jak to mohlo vyhodnotit jako `503` :)